grc/core/utils/descriptors/evaluated.py

```diff
@@ -17,8 +17,7 @@
 
 from __future__ import absolute_import
 
-import six
-
+import six, collections
```

Comment:
please don't do this, each import shall be in a separate statement. especially stdlib and 3rd party libs should be separated.

Reply:
Right! Will fix